scripts/ci/mermaid-config.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Shouldn't that be in PR with documentation?

Reply:
Indeed :)